lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -855,6 +855,100 @@ static void optimizeBatchNorm(Function *F) {
   } // For all nodes in the graph.
 }
 
+/// Check if the \p inputs of this method (vector of NodeValues) can be
+/// legal input of CreateConcat function.
+/// \returns the dimension where the Concat will take place. If not
+/// concatenatable, return -1.
+static int
+getConcatDimensionForInputs(const llvm::SmallVector<NodeValue, 16> &inputs,
```

Comment:
when you pass "SmallVector<T, size>" as a parameter, you should use this LLVM type for the parameter type:

"const llvm::SmallVectorImpl<T> &inouts". This way, you can pass a SmallVector with any "size" parameter.

Reply:
done. thx